README.md

```diff
@@ -25,7 +25,8 @@ __Update 31 Mar 2021__: Currently there are __126 tips__ divided into 14 section
 
 ## DB Models and Eloquent
 
-⬆️ [Go to top](#laravel-tips) ➡️ [Next (Models Relations)](#models-relations)
+⬆️ [Go to top](#laravel-tips
+➡️ [Next (Models Relations)](#models-relations)
```

Comment:
@Oxicode good catch with "OnUpdate", but could you fix this line? I think you made this line edit unintentionally.
Then I would merge the Pull Request.

Thanks!